R/gadgets.R

```diff
@@ -0,0 +1,58 @@
+#' Open dataset selector
+#'
+#' This function launches a shiny gadget which allows you to navigate your Crunch
+#' projects and datasets. This is useful if you can't remember a dataset's project
+#' and also saves typing long dataset names.
+#'
+#' @inheritParams listDatasets
+#' @return A `listDatasets()` call is pasted into your RStudio session`
+#' @export
```

Comment:
Is it necessary to export this function? Seems that the preferred way to get to it is through `listDatasets()` and not calling it directly.